integrationos-api/src/logic/oauth.rs

```diff
@@ -189,17 +192,25 @@ async fn oauth_handler(
         })?;
 
     let conn_definition = get_conn_definition(&state, &payload.connection_definition_id).await?;
+    let group = Uuid::new_v4().to_string().replace('-', "");
+    let namespace = "default".to_string();
+    let name = match user_event_access.environment {
+        Environment::Test => format!("{} sandbox account", conn_definition.name),
```

Comment:
DITTO. Optionally if this is repeated perhaps we could put this in a single place?